docs/features.md

```diff
@@ -53,41 +56,100 @@
+* genesis reached as described in "A successful genesis ceremony"
 * `validatorC` joins p2p, pointing on `ceremonyMaster` as bootstrap
 * `validatorC` sends `ApprovedBlockRequest` to `ceremonyMaster`
 * `ceremonyMaster` sends `ApprovedBlock` to `validatorC`
 * `validatorC` transitions to `ApprovedBlockReceivedHandler`
 * `validatorC` tip points to block (genesis) where it has no parent and Bonds holds `validatorA` and `validatorB`
 
-## Proof of stake consensus
-
-### UC: As a dApp developer I want to be able to deploy my rholang contract to a validator
-
-#### AC: A correct contract gets deployed successfully
-
-##### test: [todo: requires integration test]
-
+## Wallets
+### As a user, I want to be able to create a wallet so that I can store REV in it
+### As a user, I want to be able to add REV to my wallet so that I have available REV to pay for goods/services
+### As a user, I want to be able to remove REV from my wallet so that I can pay for goods/services
+### Expose purses inside a wallet
+### As a wallet application, I want to query a wallet contract (or the blocks) for the history of all Rev transfers to/from it
+### As a REV holder, I can move some of my REV to the control another user’s public key (or address) via a co-op supplied dApp wallet
+### As a recipient of REV (other than Genesis REV), I can use a co-op supplied dApp to view my REV balance
+### As a validator, I can move Rev to/from the key-pair for one validator node to the key-pair for another validator node or that of the co-op supplied wallet dApp
+### As a wallet dApp developer, I want to use Ethereum-style addresses for send transactions to specify the recipient, so that a) I can reuse available Ethereum utility libraries; b) so the QR code is smaller and thus faster to scan than it would be for a full public key; c) it is easier for users to verbally confirm their address versus public key; and d) so RChain is more palatable for the Ethereum community
+### As a wallet dApp developer, I want to discover and connect to rNodes that support a particular version (release number and hash) and have a minimum number of active connections, so that user risks due to interoperability issues and sybil actors are minimized
+### As a wallet user, I need a command line interface for interacting with wallets.
+### As a dApp organization, I need to have multiple approvers for any send transaction.
+## Storage
+### As a user, I want to be able to store transaction data on the blockchain so that it is available and accessible to users
+### As a user, I want to be able to store non-transaction data on the blockchain so that it is available and accessible to users
+## Bonding/Unbonding
+### As a Node Validator, I want to be able to add my stake to the network and be recognized as a validator so I can participate in proof of stake consensus and be eligible to earn rewards (validating)
```

Comment:
`As a Node Validator` <- bad language. `As a Node that wants to become a Validator`?

Reply:
My vote is for `As node operator who wants to become a validator...`